### DISCUSSION TITLE
Why isn't there an option for U2F hardware in the mandatory 2FA sign-up?

### DISCUSSION THREAD

**Discussion by ekimber** [community/community#78020] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The only reason I don't have 2FA on my account already is because U2F is only available as a 'secondary' method. I use Yubikey for 2FA, I do not use apps or SMS, which is insecure anyway. Yes I could lose the key and that's why I have a second key as a backup.  Since the code has been implemented, what exactly is the problem with enabling it as a primary method?

**Comment by Khushiyant** [comment]


**Reply by ekimber** [reply]
Sorry man but your AI answer is wack
